README.md

```diff
@@ -80,21 +80,21 @@ The `homepage:`, `author:`, `maintainer:`, and `doc:` fields are
 also used to generate the package entry.
 
 The JSON file is generated during continuous integration and
-[copied to the website](https://coq.inria.fr/opam/coq-packages.json).
+[copied to the website](https://rocq-prover.org/opam/coq-packages.json).
 JavaScript code on the website then loads it to dynamically generate
 the content of the website on the client side.
 
-See also [CEP3](https://github.com/coq/ceps/blob/master/text/003-opam-metadata.md) and
-the [deployed website](https://coq.inria.fr/opam/www/).
+See also [CEP3](https://github.com/rocq-prover/rfcs/blob/master/text/003-opam-metadata.md) and
+the [Rocq package index](https://rocq-prover.org/packages).
 
 ## Continuous integration
 
 Incoming pull requests are tested on GitLab CI. **@coqbot** pushes any opened
 or synchonized pull request to a branch named `pr-<number>` on GitLab. It will
 trigger a CI build. If the CI build runs for too long and times out, any
-member of the Coq organization of GitLab can start it again using the "Run
-Pipeline" green button at <https://gitlab.com/coq/opam-coq-archive/pipelines>.
-This will then build only on runners without pre-set timeouts (the Coq Pyrolyse
+member of the Rocq organization of GitLab can start it again using the "Run
+Pipeline" green button at <https://gitlab.com/rocq-prover/opam/-/pipelines>.
```

Comment:
This part is not current since the repo is using the GitLab Inria instance (and the organization there is still called Coq for now).